src/ConcertDisplay.js

```diff
@@ -1,15 +1,19 @@
 import React, { Component } from 'react';
 import './ConcertDisplay.css';
+import Concert from './Concert.js';
 
 export default class ConcertDisplay extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
```

Comment:
Not sure if we need to pass in props here. Since it's a class we should just be able to call `this.props` and it should access it without having it passed into the constructor.

Reply:
was wondering the same thing to be honest